python/knext/knext/builder/operator/builtin/deepke_prompt.py

```diff
@@ -0,0 +1,370 @@
+                    spg_record.upsert_property(relation_en, object_value)
+                else:
+                    s_type_zh, o_type_zh = self.relations_mapper.get(relation_zh, [None, None])
+                    if s_type_zh is None or o_type_zh is None:
+                        print(f"Unrecognized relation: {relation_zh}")
+                        continue
+                    s_type_en, _ = self.spg_type_schema_info_zh[s_type_zh]
+                    spg_record = SPGRecord(s_type_en).upsert_properties({"id": s_zh, "name": s_zh})
+                    relation_en, _, object_type = self.relation_info_zh[s_type_zh][relation_zh]
+                    spg_record.upsert_relation(relation_en, object_type, object_value)
+                spg_records.append(spg_record)
+        return spg_records
+
+    def _render(self):
+        spo_list = []
+        for spg_type in self.spg_types:
+            type_en, _ = self.spg_type_schema_info_zh[spg_type]
+            for v in spg_type.properties.values():
+                spo_list.append({'subject_type':spg_type.name_zh, 'predicate':v.name_zh, 'object_type':'文本'})
+                self.properties_mapper[v.name_zh] = spg_type
+            for v in spg_type.relations.values(): 
+                _, _, object_type = self.relation_info_en[type_en][v.name]
+                spo_list.append({'subject_type':spg_type.name_zh, 'predicate':v.name_zh, 'object_type':object_type})
+                self.relations_mapper[v.name_zh] = [spg_type, object_type]
+        self.schema_list = self.multischema_split_by_num(self.split_num, spo_list)
+
+
+
+class DeepKE_REPrompt(DeepKE_SPOPrompt):
+    template_zh: str = "你是专门进行关系抽取的专家。请从input中抽取出符合schema定义的关系三元组，不存在的关系返回空列表。请按照JSON字符串的格式回答。"
+    template_en: str = "You are an expert in relationship extraction. Please extract relationship triples that match the schema definition from the input. Return an empty list for relationships that do not exist. Please respond in the format of a JSON string."
+
+    def __init__(
+        self,
+        relation_types: List[SPGTypeName],
+        language: str = 'zh',
+        with_description: bool = False,
+        split_num: int = 4,
+    ):
+        super().__init__(relation_types, language, with_description, split_num)
+
+
+    def _render(self):
+        re_list = []
+        for spg_type in self.spg_types:
+            type_en, _ = self.spg_type_schema_info_zh[spg_type]
+            for v in spg_type.properties.values():
+                re_list.append(v.name_zh)
+                self.properties_mapper[v.name_zh] = spg_type
+            for v in spg_type.relations.values(): 
+                v_zh, _, object_type = self.relation_info_en[type_en][v.name]
+                re_list.append(v.name_zh)
+                self.relations_mapper[v.name_zh] = [spg_type, object_type]
+        self.schema_list = self.multischema_split_by_num(self.split_num, re_list)
+
+
+
+class DeepKE_KGPrompt(DeepKEPrompt):
```

Comment:
DeepKE_KGPrompt -> DeepKENEPrompt?  take letters N and E of `eNtity Extract` to avoid duplication with EE